sources/packages/backend/libs/sims-db/src/entities/application.model.ts

```diff
@@ -426,6 +427,10 @@ export interface ApplicationData {
    * Course Details is present only for Part Time course intensity.
    */
   courseDetails?: CourseDetails[];
+  /**
+   * Application PD/PPD Status.
+   */
+  applicationPDPPDStatus?: string;
 }
```

Comment:
I think we can use enum `ApplicationDisabilityStatus` instead of string here. Please let me know your thoughts.

Reply:
Thanks for making the change.